build.sbt

```diff
@@ -30,7 +30,7 @@ tag match {
 }}
 
 publishTo := {
-  val artifactory = sys.env.get("ARTIFACTORY_SERVER").getOrElse("http://artifactory.registered-traveller.homeoffice.gov.uk/")
+  val artifactory = sys.env.get("ARTIFACTORY_SERVER").getOrElse("https://artifactory.registered-traveller.homeoffice.gov.uk/")
```

Comment:
Need to change this to artifactory.digital.homeoffice.gov.uk

Reply:
fixed